frontend/src/components/pages/Scheduling/ConfirmDetails.tsx

```diff
@@ -331,7 +334,7 @@ const ConfirmDetails = ({
         </Text>
         <Text textStyle="mobileBody">{currentDonor.businessName}</Text>
       </Box>
-      {isBeingEdited && (
+      {isBeingEdited && authenticatedUser?.role === Role.DONOR && (
```

Comment:
admin should have the ability to delete recurring donations/single donations
this line should be reverted